src/spyglass/common/populate_all_common.py

```diff
@@ -14,51 +16,57 @@
 from spyglass.common.common_nwbfile import Nwbfile
 from spyglass.common.common_session import Session
 from spyglass.common.common_task import TaskEpoch
+from spyglass.common.common_usage import InsertError
+from spyglass.spikesorting.imported import ImportedSpikeSorting
```

Comment:
I was getting circular import problems with this here before

Reply:
@edeno - Sorry for the delay. I moved this import back